code/__DEFINES/mobs.dm

```diff
@@ -97,7 +96,6 @@
 #define	FLAG_APID			(1<<4)
 #define FLAG_MOTH			(1<<5)
 #define FLAG_LIZARD			(1<<6)
-#define FLAG_FELINID		(1<<7)
 #define FLAG_OOZELING		(1<<8)
 #define FLAG_FLY			(1<<9)
 #define FLAG_DEBUG_SPECIES	(1<<10)
```

Comment:
Probably need to shift the rest and not leave a gap in the middle

Reply:
done